packages/loader/container-loader/src/loader.ts

```diff
@@ -202,10 +203,60 @@ export interface ILoaderProps {
 	readonly protocolHandlerBuilder?: ProtocolHandlerBuilder;
 }
 
+/**
+ * Other props which are used to load a container on top of the basic loader props.
+ * @alpha
+ * @legacy
+ */
+export interface ILoaderResolveContainerProps extends ILoaderProps {
```

Comment:
```suggestion
export interface IResolveContainerProps extends ILoaderProps {
```

Reply:
let's remove ILoader from these, as we will probably want to deprecate that as well eventually